crates/ruff_python_parser/src/lexer.rs

```diff
@@ -428,10 +429,46 @@ impl<'source> Lexer<'source> {
                     self.cursor.bump();
                     value.push('\\');
                 }
+                '?' => {
+                    self.cursor.bump();
+                    let mut question_count: u8 = 1;
```

Comment:
1 because the first char is already `?`, so we'll bump it up.

Reply:
Lt's use a `u32` here, just to be safe
```suggestion
                    self.cursor.bump();
                    let mut question_count = 1u32;
```